CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
+helps, and credit will always be given. You can contribute in the ways listed below.
+
+## Report Bugs
+
+Report bugs using GitHub issues.
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+## Fix Bugs and Implement Features
+
+Look through the GitHub issues for bugs and features. Anything tagged with "bug", "help
+wanted" or "enhancement" is open to whoever wants to implement it.
+
+## Write Documentation
+
+The ORCESTRA book could always use more documentation, whether through describing a
+a sub-campaign or an instrument measurement or by adding a data processing example.
+Please feel free to add information yourself or open an issue if you lack the time 
+or knowledge to add it right away.
+
+
+## Get Started
+
+Ready to contribute? Here's how to set up the `ORCESTRA book` for local development.
+
+1. Fork the repo on GitHub.
+2. Clone your fork locally.
+3. Install your local copy into a virtualenv, e.g., using `conda`.
+4. Create a branch for local development and make changes locally.
+5. Commit your changes and push your branch to GitHub.
+6. Submit a pull request through the GitHub website.
+
+## Building the book
+
+To check your changes locally you can build the book:
+
+- (Recommended) Remove the existing `orcestra-book/_build/` directory
```

Comment:
Why is this recommended in general?
This will also remove the jupyter execution cache and thus slow down the build process a lot (especially once we have more executable content). I'd rather try to keep the `_build` folder as long as possible and only remove it if you have some suspicion that the rendered book didn't get fully updated.

In fact, we also keep the `_build` folder in the Github Actions Cache between multiple runs of the CI pipeline to speed up the rendering of the book.